js/src/forum/components/RankingsPage.js

```diff
@@ -73,10 +73,10 @@ export default class RankingsPage extends Page {
                                         <td>
                                             <div className="PostUser">
                                                 <h3 className="rankings-info">
-                                                    <Link href={app.route.user(user)}>
+                                                    <a href={app.route.user(user)}>
```

Comment:
I know, but I didn't have found any other way to "fix" the bug...

Reply:
If I'm not mistaken you need to use `force`:

```suggestion
                                                    <Link href={app.route.user(user)} force={true}>
```

or something similar, see:

```js
    // Mithril 2 does not completely rerender the page if a route change leads to the same route
    // (or the same component handling a different route).
    // Here, the `force` parameter will use Mithril's key system to force a full rerender
    // see https://mithril.js.org/route.html#key-parameter
    if (extract(attrs, 'force')) {
      if (!('state' in options)) options.state = {};
      if (!('key' in options.state)) options.state.key = Date.now();
    }
```

Reply:
@luceos no, it doesn't work...

Reply:
@maicol07 can you confirm how this doesn't work? I've just tested the above locally and it works absolutely perfectly so far as I can tell? 

This is what I ended up with : 
```
<link href={app.route.user(user)} force={true}>
                                                        {i < 4 ? avatar(user, { className: 'info-avatar rankings-' + i + '-avatar' }) : ''}{' '}
                                                        {username(user)}
                                                    </link>
```

Reply:
* please ignore the horrendous spacing - thanks Github.

Reply:
For clarity, I checked this by:

- Checked out your fixes branch locally.
- Ran: `composer update` then `php flarum migrate` then `php flarum assets:publish` then ran `php flarum cache:clear`
- When I go to the rankings page and click on any of my test users, the user profile page loads perfectly, and I do not have issues with repeated user banners.

Reply:
The issue is that we use `<a>` tags here instead of Flarum's own `<Link>` tag to "fix" the issue. If you change these `<a>` tags to `<Link>` you should see the issue.

Reply:
As discussed on Discord @davwheat that's what I did - @imorland has also seen screenshots of this working for me.
Can anyone from @FriendsOfFlarum/maintainers please confirm?

Reply:
@FriendsOfFlarum/contributors / @FriendsOfFlarum/qa also feel free to assist <3

Reply:
> For clarity, I checked this by:
> 
> * Checked out your fixes branch locally.
> * Ran: `composer update` then `php flarum migrate` then `php flarum assets:publish` then ran `php flarum cache:clear`
> * When I go to the rankings page and click on any of my test users, the user profile page loads perfectly, and I do not have issues with repeated user banners.

tried this, but didn't solve the issue

Reply:
> tried this, but didn't solve the issue

can you detail what you did? (And did you rebuild the JS? I cannot replicate this locally at all.
Can you also confirm what other extensions you have loaded. I’m leaning towards an extension compatibility issue

Reply:
@katosdev have you tried with my branch (with the `<a>` tag) or the main one (with the `<Link>` tag)?

Reply:
Both - it works fine with <link> and was confirmed also by @EvilExecutor

Reply:
So, I've checked with @rossoblu93 and it seems that Color Circles by @clarkwinkelmann was incompatible. So I'm going to change it back to `<Link>`